templates/_layouts/examples.html

```diff
@@ -37,12 +37,36 @@
           document.addEventListener('DOMContentLoaded', function() {
             var body = document.body;
             var controls = fragmentFromString('<div class="u-baseline-grid__toggle"><label class="p-switch"><input type="checkbox" class="p-switch__input js-baseline-toggle" /><span class="p-switch__slider"></span><span class="p-switch__label">Toggle baseline grid</span></label></div>');
+            var themes = fragmentFromString('<div class="u-theme-toggle"><button class="u-theme-toggle__button u-theme-toggle__dark js-dark-theme-toggle">Dark</button><button class="u-theme-toggle__button u-theme-toggle__light js-light-theme-toggle">Light</button><button class="u-theme-toggle__button u-theme-toggle__paper js-paper-theme-toggle">Paper</button></div>');
```

Comment:
I think this JS got big enough to justify a separate file.

We have JS in `templates/static/js` so maybe create a file there (`example-tools.js` or something like that) and move this whole script there, and include it from there rather then directly in HTML.